newsletter-june-2021.rst

```diff
@@ -0,0 +1,87 @@
+.. post:: June 3, 2021
+   :tags: newsletter, python
+   :author: Juan Luis
+   :location: MAD
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from last month,
+      current focus, and upcoming features from June.
+
+Read the Docs newsletter - June 2021
+====================================
+
+Welcome to a new edition of our monthly newsletter, where we
+openly share the most relevant updates around Read the Docs,
+offer a summary of new features we shipped
+during the previous month,
+and share what we'll be focusing on in the near future.
+
+Company highlights
+------------------
+
+- We applied to the next round of the CZI Essential Open Source Software for Science!
+  Our proposal is even more ambitious than the previous one,
+  and we are excited about the possibilities for our users
+  if we finally get accepted.
+- We keep making progress on the current CZI grant work:
+  `Juan Luis`_ sent the first pull request for `our new Sphinx
+  tutorial <https://github.com/sphinx-doc/sphinx/pull/9276>`_,
+  which received positive feedback from the maintainers,
+  and `Manuel`_ is about to release a new version of ``sphinx-hoverxref``
+  `with support for intersphinx <https://github.com/readthedocs/sphinx-hoverxref/pull/86>`_
+  (we will write a dedicated blog post about this soon).
+- We added a `Data Processing Agreement <https://docs.readthedocs.io/en/stable/legal/dpa/>`_
+  to our official documentation.
+- We are making progress towards a new streamlined workflow
+  that helps us while the company grows.
```

Comment:
```suggestion
- We are iterating on an improved roadmap workflow to make sure our team
   stays productive as we grow. 
```

Original feels a bit vague, should be specific it's about roadmap?

Reply:
Better now, thanks!